server/app.js

```diff
@@ -1,8 +1,12 @@
 const express = require("express")
 const app = express()
+const endpoints = require("../endpoints.json")
 
 const { getTopics } = require("./controllers/topics-controller.js")
 
+app.get("/api", (req, res, next) => {
+  res.status(200).send({endpoints: endpoints})
+})
```

Comment:
This callback function is a controller - as all your other controllers are going to be extracted into their own files, I'd suggest doing the same for this one.